app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/FeatureAttribute.ts

```diff
@@ -0,0 +1,13 @@
+// In an OOP Language -
+
+import {FilterName} from "../states/map/FilterName";
+
+// TypeScript
+
+export abstract class FeatureAttribute {
+  abstract get name(): string;
+  abstract get isNumeric(): boolean;
+  abstract get isString(): boolean;
+  abstract get filterType(): FilterName;
```

Comment:
filterName? I guess filterType is the right nomenclature, and FilterName is not right for the enum.